classes/XLite/Module/ShipStation/Api/Model/Repo/Order.php

```diff
@@ -38,10 +38,17 @@ public function getOrdersFromRenewDate($dtOrderUpdateStart = '', $dtOrderUpdateE
      */
     public function addOrderTrackingNumber($intOrderNumber = 0, $intTrackingNumber = 0) 
     {
-        \Includes\Utils\Database::execute("INSERT INTO " . \Includes\Utils\Database::getTablesPrefix() . "order_tracking_number (order_id, value, creationDate) VALUES (:order_id, :tracking_number, NOW())", array(
-            ':order_id' => $intOrderNumber,
-            ':tracking_number' => $intTrackingNumber
-        ));
+        if (\XLite\Module\ShipStation\Api\Main::getMajorVersion() == '5.4') {
```

Comment:
Is it safe to parse this as a number so that we still resolve true for `5.5` and up?

Reply:
Is anything safe in PHP? Your questions are too deep for a Monday...